.github/workflows/release_mac.yml

```diff
@@ -0,0 +1,62 @@
+name: MacRelease
+
+on:
+  push:
+    branches:
+      - rel-*
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+env:
+  MACOSX_DEPLOYMENT_TARGET: 10.9
+jobs:
+  build:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        architecture: ['x64']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: ${{ matrix.architecture }}
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade setuptools
+        pip install pytest==5.4.3 numpy wheel nbval
+        # install protobuf 3.11.3
+        brew update
+        brew install autoconf && brew install automake
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v3.11.3/protobuf-cpp-3.11.3.tar.gz
+        tar -xvf protobuf-cpp-3.11.3.tar.gz
+        cd protobuf-3.11.3
+        ./autogen.sh
+        ./configure
+        make
+        make check
+        sudo make install
+    - name: Build wheel
+      env:
+        ONNX_BUILD_TESTS: 1
+        USE_MSVC_STATIC_RUNTIME: 1
+        ONNX_ML: 1
+        CMAKE_ARGS: "-DONNX_USE_LITE_PROTO=ON -DONNXIFI_DUMMY_BACKEND=ON -DONNX_WERROR=ON"
+      run: |
+        python setup.py bdist_wheel
```

Comment:
run  git submodule update --init --recursive before line 54

Reply:
Isn't it the same as line 25?